devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/picocli-example/codestart.yml

```diff
@@ -0,0 +1,28 @@
+name: picocli-example
+ref: picocli
+type: code
+tags:
+  - example
+  - singleton-example
+language:
+  base:
+    data:
+      title: Picocli example
+      description: |
+        <p>Hello and goobye are civilization fundamentals. Lets not forget it with this picocli application by changing the <code>command</code> and <code>parameters</code>.</p>
+        <p><b>Go Subatomic!</b></p>
+        ## Dev mode
+        Also for picocli applications the dev mode is supported. When running mvn compile quarkus:dev, the picocli application is executed and on press of the Enter key, is restarted.
+
+        As command mode applications will often require arguments to be passed on the commandline, this is also possible in dev mode via:
+
+        `mvn compile quarkus:dev -Dquarkus.args='--help'`
+
+        The same can be achieved with Gradle:
+
+        `./gradlew quarkusDev --quarkus-args='--help'`
+
+      guide: https://quarkus.io/guides/picocli
```

Comment:
```suggestion
```

You can remove that part, it was used because the title and description is also used in the html for the cloud codestarts.

Reply:
Which is not the case anymore ?

Reply:
It is, but not for this example which doesn't include any web feature.